docs/articles/documentation/reference/testcafe-api/browserconnection/opened.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: docs
+title: BrowserConnection.opened Event
+permalink: /documentation/reference/testcafe-api/browserconnection/opened.html
+---
+# BrowserConnection.opened Event
+
+Fires when a browser launched by a [browser provider](../../../guides/extend-testcafe/browser-provider-plugin.md#implement-the-browser-provider) has connected to TestCafe and reported that it is ready to run tests.
+
+```js
+browserConnection.once('opened', callback)
+```
+
+If every remote browser connection does not emit this event within **6** minutes, TestCafe throws an error. You can configure this timeout with the [browserInitTimeout](../../configuration-file.md#browserinittimeout) configuration file property.
```

Comment:
"does not emit this event" Why do you use **emit**?

Reply:
[All objects that emit events are instances of the EventEmitter class](https://nodejs.org/api/events.html#events_events:~:text=All%20objects%20that%20emit%20events%20are%20instances%20of%20the%20EventEmitter%20class.)

Therefore, `BrowserConnection` is an event emitter too. 

Maybe that's too technical. If it sounds better, we could go with:

> If every remote browser connection does not fire this event within **6** minutes...

OR

> If every remote browser connection does not call this event within **6** minutes,

Reply:
If every remote browser connection does not fire this event within 6 minutes...